src/main/java/io/excaliburfrc/robot/commands/ShootBallsCommand.java

```diff
@@ -4,25 +4,27 @@
 import edu.wpi.first.wpilibj2.command.button.Trigger;
 import io.excaliburfrc.lib.RepeatingCommand;
 import io.excaliburfrc.robot.subsystems.Intake;
+import io.excaliburfrc.robot.subsystems.LEDs;
 import io.excaliburfrc.robot.subsystems.Shooter;
 
 public class ShootBallsCommand extends ParallelDeadlineGroup {
   /**
    * @param trigger an additional trigger, will release a ball only when this (and the velocity
    *     check) are reached.
    */
-  public ShootBallsCommand(Intake intake, Shooter shooter, Trigger trigger) {
+  public ShootBallsCommand(Intake intake, Shooter shooter, LEDs leds, Trigger trigger) {
     super(
         // FIXME: handle mechanism failures? (ie stuck balls, etc)
         new WaitUntilCommand(intake::isEmpty),
         shooter.accelerateFenderCommand(),
         new RepeatingCommand(
             new SequentialCommandGroup(
                 new WaitUntilCommand(new Trigger(shooter::isAtTargetVelocity).and(trigger)),
-                intake.shootBallCommand())));
+                intake.shootBallCommand())),
+        LEDs.getInstance().setColorCommand(LEDs.LedMode.BLUE));
```

Comment:
if you're getting the LEDs instance as a parameter, there's no reason to use `getInstance()` here: use the instance you got.

Reply:
That's what I meant... There's no need to get it as a parameter and use the static getInctance